src/redux/store.ts

```diff
@@ -1,9 +1,13 @@
 import { configureStore } from "@reduxjs/toolkit"
+import drawerReducer from "./drawer/drawerSlice"
+import oauthReducer from "./oauth/oauthSlice"
 import taskReducer from "./task/taskSlice"
 
 const store = configureStore({
   reducer: {
     allTasks: taskReducer,
+    allOauth: oauthReducer,
+    allDrawer: drawerReducer,
```

Comment:
💭 The names of these parts of state seem a bit odd to me. Consider something like:
```suggestion
    tasks: taskReducer,
    auth: oauthReducer,
    drawer: drawerReducer,
```